static/src/javascripts/projects/common/modules/audio/AudioPlayer.js

```diff
@@ -357,12 +365,16 @@ export default class AudioPlayer extends Component<Props, State> {
             checkForTimeEvents(this.props.mediaId, percentPlayed);
         }
 
-        this.incrementBlock(this.audio.currentTime);
-
-        this.setState({
-            currentTime: this.audio.currentTime,
-            currentOffset: percentPlayed,
-        });
+        // pause when it gets to the end
+        if (this.audio.currentTime > this.state.duration - 1) {
```

Comment:
I think we decided this can be half a second?
Also worth noting here that this is to work around a chrome bug